src/Header.svelte

```diff
@@ -0,0 +1,100 @@
+<script>
+  import { tasks } from "./stores";
+
+  // buttons are so 2000's, let's use emojis instead
+  import EmojiButton from "./EmojiButton.svelte";
+
+  // tooltips are helpful
+  import ToolTip from "./ToolTip.svelte";
+
+  let taskDescription = "";
+
+  // string to hold the title
+  $: title = selectRandomTitle();
+
+  // function to select a random title
+  function selectRandomTitle() {
+    const titles = [
+      "To-Do List",
+      "Aufgabenliste",
+      "Lista de Pendientes",
+      "Liste de Choses à Faire",
+      "Lista di Cose da Fare",
+      "Lista de Afazeres",
+      "Список Дел",
+      "待办事项",
+      "タスク一覧",
+      "רשימת המטלות",
+      "🍈🍈🍈", //get it, honeydew? honey do? i'm hilarious...
+    ];
+    const index = Math.floor(Math.random() * titles.length);
+    return titles[index];
+  }
+
+  // function to add task if users presses enter, instead of clicking the button
+  const onKeyDown = (event) => {
+    if (event.key === "Enter") {
```

Comment:
It's much easier to use [`event.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) or [`event.code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) instead of `charCode` because you can actually remember what the key is without having to add a comment.